### PR TITLE
Add New Port Frogger 2

### PR DESCRIPTION
**Pull request by MadShmupper**
Add New Port Frogger 2

# New Port for { Frogger 2 }

## Game Information
- **Title**: { Frogger 2 }
- **URL**: { https://ryan-silberman.itch.io/frogger2 }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/frogger2/gameinfo.xml
Comment (tabreturn) [inline ports/frogger2/gameinfo.xml:1]: Missing space after stop --
frog children.A full-bown

File: ports/frogger2/screenshot.jpg
Comment (tabreturn) [inline ports/frogger2/screenshot.jpg:1]: I possess an irrational hatred of JPG artefacts :) Here are some PNG screenshots if you prefer -- 
screenshot1
screemshot2

**Review by tabreturn — DISMISSED** [review]

File: ports/frogger2/gameinfo.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Frogger 2.sh</path>
+    <name>Frogger 2</name>
+    <desc>Play as Frogger or Lillie Frog as they conquer 32 stages to rescue the missing frog children.A full-blown fan remake of the Game Boy Color version of Frogger 2!</desc>
```
Comment (Cebion) [inline ports/frogger2/gameinfo.xml:6]:
```suggestion
    <desc>Play as Frogger or Lillie Frog as they conquer 32 stages to rescue the missing frog children. A full-blown fan remake of the Game Boy Color version of Frogger 2!</desc>
```

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/frogger2/gameinfo.xml (hunk 1 above)
Comment (MadShmupper) [reply ports/frogger2/gameinfo.xml]: Fixed the space.

File: ports/frogger2/screenshot.jpg
Comment (MadShmupper) [reply ports/frogger2/screenshot.jpg:1]: Ty. I'll have a look.

File: ports/frogger2/gameinfo.xml
Comment (MadShmupper) [reply ports/frogger2/gameinfo.xml:1]: Fixed🙂

**Review by Cebion — DISMISSED** [review]

File: ports/frogger2/Frogger 2.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        rm "gamedata/data.win"
+    fi
+fi
+[ -f "./gamedata/data.win" ] && mv gamedata/data.win gamedata/game.droid
+
+$GPTOKEYB "gmloader" -c "./frogger2.gptk" &
+echo "Loading, please wait... " > /dev/tty0
+
+$ESUDO chmod +x "$GAMEDIR/gmloader"
+pm_platform_helper $GAMEDIR/gmloader
```
Comment (cdeletre) [inline ports/frogger2/Frogger 2.sh:61]:
```suggestion
pm_platform_helper "$GAMEDIR/gmloader"
```

**Review by Cebion — APPROVED** [review]

**Review by cdeletre — APPROVED** [review]

**Pull request merged**